tests/ImageSharp.Tests/Color/ColorTests.CastTo.cs

```diff
@@ -88,6 +88,19 @@ public void Bgr24()
                 Bgr24 data = color;
                 Assert.Equal(source, data);
             }
+
+            [Fact]
+            public void TPixel()
+            {
+                var source = new RgbaVector(1, .1F, .133F, .864F);
```

Comment:
I recommend to stress the precision in this method:

```suggestion
                var source = new RgbaVector(float.Epsilon, 2 * float.Epsilon, float.MaxValue, float.MinValue);
```